packages/react-native/gradle/libs.versions.toml

```diff
@@ -8,7 +8,7 @@ ndkVersion = "27.1.12297006"
 # Dependencies versions
 agp = "8.8.0"
 androidx-annotation = "1.6.0"
-androidx-appcompat = "1.6.1"
+androidx-appcompat = "1.7.0"
```

Comment:
This change needs to land in isolation (in a separate commit). Could you do so?

Reply:
Separate PR as needs to be imported/landed separately + is worth a separate changelog entry

Reply:
https://github.com/facebook/react-native/pull/49594